### PR TITLE
Add Matter livestream

### PR DESCRIPTION
**Pull request by GuySie**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by marcelveldt — DISMISSED** [review]
One comment I have is if the event shouldn't be preregistered and linked in the article ?

File: source/_posts/2024-01-04-matter-livestream.markdown [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "The State of Matter livestream"
+description: "We’re having a Matter livestream on Wednesday, January 10. Ask all your questions about Matter and Home Assistant in the comment section!"
+date: 2024-01-04 00:00:01
+date_formatted: "January 4, 2024"
```
Comment (frenck) [inline source/_posts/2024-01-04-matter-livestream.markdown]:
```suggestion
date: 2024-01-05 00:00:01
date_formatted: "January 5, 2024"
```

**Review by frenck — CHANGES_REQUESTED** [review]
The filename needs adjustments too

**Review by frenck — APPROVED** [review]
Thanks, @GuySie 👍 

../Frenck

**Pull request merged**